enki/PhysicalEngine.cpp

```diff
@@ -1199,6 +1220,9 @@ namespace Enki
 		// interact objects with walls and control step
 		for (ObjectsIterator i = objects.begin(); i != objects.end(); ++i)
 		{
+			if(SkipPhysicsObjectsList.find(*i) != SkipPhysicsObjectsList.end())
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.